test/gql/test_filter.py

```diff
@@ -630,19 +630,19 @@ def test___str__(self):
             result,
         )
 
-        test_filter = helper_get_test_filter("valueString", "this is an escape sequence \a")
-        result = str(Where(test_filter))
-        self.assertEqual(
-            'where: {path: ["name"] operator: Equal valueString: "this is an escape sequence \\u0007"} ',
-            result,
-        )
-
-        test_filter = helper_get_test_filter("valueString", "this is a hex value \u03A9")
-        result = str(Where(test_filter))
-        self.assertEqual(
-            'where: {path: ["name"] operator: Equal valueString: "this is a hex value \\u03a9"} ',
-            result,
-        )
+        # test_filter = helper_get_test_filter("valueString", "this is an escape sequence \a")
```

Comment:
can you readd some tests? Or are they not relevant anymore?

Reply:
The change was the least amount of work for the most gain in time for the newest release

Reply:
and we don't need special tests here, because we use the same sanitation for BM25 etc, right?

Reply:
Yes, all user-defined strings in queries are sanitised by the same regex function

Reply:
perfect!